src/Translation/locales/es.json

```diff
@@ -405,6 +414,18 @@
         "Puedes usar navegadores móviles como {coinbase_link} o {imtoken_link}."
     }
   },
+  "terms_modal": {
+    "also_agree":
+      "También acepta nuestro {privacy_policy_link}, que describe cómo procesamos su información.",
+    "by_agreeing":
+      "Al seleccionar \"Acepto\" a continuación, acepta nuestro {terms_of_service_link}.",
+    "explanation":
+      "Este dApp interactúa con un intercambio descentralizado que se ejecuta en la cadena de bloques Ethereum.",
```

Comment:
`***Esta***...cadena de bloques ***de*** ....`

Maybe we should keep `intercambio` as `exchange`